indico/modules/events/abstracts/models/persons.py

```diff
@@ -58,5 +58,5 @@ def locator(self):
 
     @return_ascii
     def __repr__(self):
-        return format_repr(self, 'id', 'person_id', 'abstract_id', is_speaker=False, author_type=AuthorType.none,
+        return format_repr(self, 'id', 'person_id', 'abstract_id', is_speaker=False, author_type=None,
```

Comment:
Do you really want to show the author type in the repr even if they aren't an author?